team_admin/discord.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Discord Server
+---
+
+# Discord Server
+
+To assist with communications, especially in a virtual competition, there's a [Discord](https://discord.com/) communication both for communication with blue-shirts and among your team.
+
+If you do not wish to, or are unable to join Discord, you can still get support from <{{ site.emails.teams }}> as normal.
+
+## Joining
+
+Invite links will be sent to team leaders. Only competitors should be given access.
+
+Once invited, you'll only have access to a few channels, namely `#welcome-and-rules` and `#welcome-<username>` (with your username in). Before you can join, you must read the rules shown in the `#welcome-and-rules` channel, and enter the password for your team in your personal `#welcome-<username>` channel.
+
+## Channels
+
+The server has a number of channels for specific conversation. Please try to keep channels on-topic, and move channels if the conversation tangents. Each channel has a topic at the top of the screen showing what it's about.
+
+### Team channels
+
+Along with the common channels, there are some channels visible only to members of your team.
+
+`team-ABC` (where `ABC` is your TLA) is a channel only your team (and blue-shirts) can see. Here you can converse among your team, and get help from blue-shirts without giving away too much of your strategy. These channels should be considered your own, and can be used to discuss anything competition related for your team.
```

Comment:
Are these private to that team? If not, I suggest saying so explicitly as I suspect that teams could easily assume that they are.

Separately, should we provide teams with a private channel? We could presumably set these up so that their team leaders and any volunteer could access it (which I think covers the public/visibility side), but could be useful to the teams.
Feels like this might help teams feel they can discuss their code/strategy with a volunteer more freely, which is the main thing I've always perceived as the reason people don't like using the forums.

Reply:
They are private to teams, yes. The only people who can see those channels are members of that team (competitor or team leader), and blueshirts. 

That's the point of this section (see previous line).

Reply:
Turns out: I can't read.